providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/config.py

```diff
@@ -26,7 +26,7 @@ class CacheType(Enum):
 DEFAULT_RESOLVER_TYPE = ResolverType.RPC
 DEFAULT_RETRY_BACKOFF = 1000
 DEFAULT_RETRY_BACKOFF_MAX = 120000
-DEFAULT_RETRY_GRACE_ATTEMPTS = 5
+DEFAULT_RETRY_GRACE_PERIOD = 5
```

Comment:
what kind of unit is it? can you add a short comment?

Reply:
Thank you added time unit to default, and to env var - maybe we need to rethink the spec.

@toddbaert wdyt should we add a unit to the spec var name?